### PR TITLE
add zolar company

### PR DESCRIPTION
**Pull request by Wasiq08**
This is a modified version of, and should adhere to, the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

* [ ] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply**
* [ ] You are an employee of the company mentioned and confirm all included details are correct.
* [ ] You know your alphabet - company is listed in alphabetical order in the README.md file.
* [ ] The company directly hires employees. ** No bootcamps / freelance sites / etc**
* [ ] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
* [ ] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
* [ ] An entry has been added or edited within the [readme.md file](../../main/README.md).
* [ ] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
* [ ] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
* [ ] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

**Comment by Wasiq08** [comment]
@dougaitken please review and merge

**Comment by dougaitken** [comment]
@Wasiq08 Please be patient, it's been open less than an hour. I get a ping for every open PR so I've not missed it. There's something I need to fix first before this can be merged.

File: README.md [hunk 1]
```diff
@@ -617,4 +617,5 @@ Name | Website | Region
 [Zeit.io](/company-profiles/zeit-io.md) | https://zeit.io/ | Germany, The Netherlands, Spain, Chile
 [Zignaly.com](/company-profiles/zignaly.md) | https://zignaly.io/ | Worldwide
 [Zip.co](/company-profiles/zip-co.md) | https://zip.co/us/ | USA, Canada
+[Zolar](/company-profiles/zolar.md) | https://www.zolar.de/ | Germany
```
Comment (dougaitken) [inline README.md:620]: This says Germany but profile says Europe, please clarify.

File: company-profiles/zolar.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+- PostgreSql
+
+Queues:
+
+- RabbitMQ
+- Redis
+
+## Office locations
+
+**None**
```
Comment (dougaitken) [inline company-profiles/zolar.md:54]: Please delete if not applicable

**Review by dougaitken — CHANGES_REQUESTED** [review]

**Review by dougaitken — APPROVED** [review]

**Review by dougaitken — APPROVED** [review]

**Comment by dougaitken** [comment]
Thanks for contributing @Wasiq08 it is appreciated

**Pull request merged**